cli/wallet/wallet.go

```diff
@@ -542,24 +555,53 @@ func importMultisig(ctx *cli.Context) error {
 		}
 	}
 
-	var label *string
 	if ctx.IsSet("name") {
 		l := ctx.String("name")
 		label = &l
 	}
-	acc, err := newAccountFromWIF(ctx.App.Writer, ctx.String("wif"), wall.Scrypt, label, pass)
+
+	found := false
+	for _, pub := range pubs {
+		for _, wallAcc := range wall.Accounts {
+			if wallAcc.ScriptHash().Equals(pub.GetScriptHash()) {
+				if !found {
+					acc = wallAcc
```

Comment:
Ref. https://github.com/nspcc-dev/neo-go/pull/3293#discussion_r1467394095.

This error `address 'NSi1CEjpiEk33TqoQ6UvqaW1sB34mLiKk5' is already in wallet` is because we need to create a copy of `wallAcc` here.